doc/dev/custom_devices.rst

```diff
@@ -181,3 +186,145 @@ After doing so, Catalyst should be able to interface with your custom device.
     @qml.qnode(CustomDevice(wires=1))
     def f():
         return measure(0)
+
+
+Below is an example configuration file
```

Comment:
Is this accurate?
```suggestion
Below is an example configuration file with inline descriptions of how to fill out the fields. All
headers and fields are generally required, unless stated otherwise.
```